website/www/site/content/en/contribute/_index.md

```diff
@@ -300,10 +347,9 @@ export GOLANG_PROTOBUF_REGISTRATION_CONFLICT=ignore
 
    Use `R: @username` in the pull request to notify a reviewer.
 
-1. If you don't get any response in 3 business days, email the [dev@ mailing list](/community/contact-us) to ask for someone to look at your pull
-   request.
+1. If you don't get any response in 3 business days, email the [dev@beam.apache.org mailing list](/community/contact-us) to ask for someone to look at your pull request.
```

Comment:
Pls change link to : https://beam.apache.org/community/contact-us/

Reply:
done